src/goose/specs.clj

```diff
@@ -108,8 +108,10 @@
 
 ;;; ============== Batch ==============
 (s/def ::callback-fn-sym (s/nilable ::fn-sym))
+(s/def ::linger-in-hours nat-int?)
```

Comment:
Can we keep it `linger-sec` like other configurations? Users can be told it's set it to 86400 by default (Experienced devs will understand 86400 is 24 hours, don't need to label it in hours I feel.)

No strong opinions here, just aiming for consistency with other config which are in `sec`

Reply:
Made this `linger-sec`. We discussed that this is more granular and would provide more flexibility to users.